src/core/types/Query.ts

```diff
@@ -263,3 +271,40 @@ export interface GetBorrowerPositionsResponse extends BaseLineFragResponse {
     };
   };
 }
+
+//TO-DO: change the schema to meet criteria
+export interface LenderPositionsResponse {
+  positions?: LinePageCreditFragResponse[];
+}
+
+export interface PositionsResponse {
```

Comment:
@Seroxdesign I'm planning to delete the `GetBorrowerPositionsResponse` once I've confirmed that the `state.user.portfolio` object suits your needs.